release-notes/opensearch-dashboards-reports.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,40 @@
+### **Version 1.2.0.0 Release Notes**
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.2.0
+
+### **Bug Fixes**
+* Fix quoting and url-encoding ([#153](https://github.com/opensearch-project/dashboards-reports/pull/153))
+* Remove hard coded localhost when calling API ([#172](https://github.com/opensearch-project/dashboards-reports/pull/172))
+* rename plugin helper config file name to consistent with OSD ([#180](https://github.com/opensearch-project/dashboards-reports/pull/180))
+* Support range filters for csv reports ([#185](https://github.com/opensearch-project/dashboards-reports/pull/185))
+* Fix Report Creation after Report Definition Creation ([#196](https://github.com/opensearch-project/dashboards-reports/pull/196))
+* fix csv missing fields issue and empty csv on _source fields ([#206](https://github.com/opensearch-project/dashboards-reports/pull/206))
+
+
+### **Infrastructure**
+* Taking RBAC settings from Alerting plugin default to false ([#165](https://github.com/opensearch-project/dashboards-reports/pull/165))
+* Integrate notifications backend ([#129](https://github.com/opensearch-project/dashboards-reports/pull/129))
+* Build email message from template with reports links ([#184](https://github.com/opensearch-project/dashboards-reports/pull/184))
+* Use advanced settings for date format in csv reports ([#186](https://github.com/opensearch-project/dashboards-reports/pull/186))
+* Remove Notifications from Create/Edit ([#212](https://github.com/opensearch-project/dashboards-reports/pull/212))
+* Remove calling notifications in reports scheduler ([#211](https://github.com/opensearch-project/dashboards-reports/pull/211))
```

Comment:
i think you can put them all in `Enhancements`, infrastructure is mostly unit/integration tests related PRs

Reply:
ohh got it. I'll change accordingly